cypress/support/setup.ts

```diff
@@ -142,18 +142,22 @@ export type SkipOptions =
 export const setupPod = (
   user: UserConfig,
   community: CommunityConfig,
-  options?: {
-    skip: SkipOptions[]
+  {
+    skip = [],
+    hospexContainerName = 'test-community',
+  }: { skip: SkipOptions[]; hospexContainerName: string } = {
+    skip: [],
+    hospexContainerName: 'test-community',
```

Comment:
This looks to me like a repeated initialisation.

Reply:
Thank you @mariha for the careful review! My initial issue was with TypeSript typing. After a little research, this was indeed straightforward to fix. ac5e589d52f203ac28e65febc489a5f5610ad066